test/test_sparse.py

```diff
@@ -64,7 +64,7 @@ def _op_supports_any_sparse(op):
 
 CUSPARSE_SPMM_COMPLEX128_SUPPORTED = (
     IS_WINDOWS and torch.version.cuda and version.parse(torch.version.cuda) > version.parse("11.2")
-) or (not IS_WINDOWS and not TEST_WITH_ROCM)
+) or (not IS_WINDOWS)
```

Comment:
Can you please update this condition to check for torch.version.hip for 6.0.0 on ROCm.